src/modules/projects/components/tables/ProjectHouseholdsTable.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import { Card } from '@mui/material';
```

Comment:
We could add more stories here to test different cases, but I was thinking to leave it as is and create more fleshed-out stories for `GenericTable` with 6761, or later. Let me know what you think.